Mathlib/Logic/Equiv/Fin.lean

```diff
@@ -334,6 +334,10 @@ theorem coe_finRotate (i : Fin n.succ) :
     (finRotate n.succ i : ℕ) = if i = Fin.last n then (0 : ℕ) else i + 1 := by
   rw [finRotate_succ_apply, Fin.val_add_one i]
 
+theorem lt_finRotate_of_ne_last {i : Fin (n + 1)} (hi : i ≠ Fin.last n) :
+    i < finRotate _ i := by
```

Comment:
Can this be promoted to an `Iff`? Also, is something like `(finRotate _).symm i < i` similarly true for `i ≠ 0`?

Reply:
Added a couple of lemmas and changed the theorem to an iff statement.